celery/beat/__init__.py

```diff
@@ -26,14 +26,14 @@
 from kombu.utils import cached_property, reprcall
 from kombu.utils.functional import maybe_evaluate
 
-from . import __version__
-from . import platforms
-from . import signals
-from .five import items, reraise, values, monotonic
-from .schedules import maybe_schedule, crontab
-from .utils.imports import instantiate
-from .utils.timeutils import humanize_seconds
-from .utils.log import get_logger, iter_open_logger_fds
+from .. import __version__
```

Comment:
The celery source code actually never uses more than one level of relative imports (so no more than one dot).
It used to, but it ended up being a mess